tests/src/Common/PerfHarness/PerfHarness.csproj

```diff
@@ -6,9 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.performance.api">
-      <Version>1.0.0-beta-build0006</Version>
-    </PackageReference>
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0007" />
```

Comment:
Yes you can still do an import the same way as you used to be able to `<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />`